app/experimenter/experiments/tests/test_models.py

```diff
@@ -1043,6 +1043,65 @@ def test_experiment_firefox_channel_sort_does_sorting(self):
             ],
         )
 
+    def test_clone(self):
```

Comment:
Yeah this is a bit tricky, in some senses we should check literally every single field here, but the test would be super long, and also super fragile because we'd have to update it for any/every field change we land so, i'm on the fence about how explicit to make this test. @tiftran what do you think?

Reply:
could we utilize something like the ```set_to_none_fields``` and have a loop through the fields to assert that they're ```None```. That would maybe reduce the amount of future updating of this test? Other than that, I'm not too sure, but I agree this test would be super long if we check every field...

Reply:
After some thought, I think we should just test some fields, not all, for now.